DataFormats/SiStripDetSetCompressedCluster/src/SiStripDetSetCompressedCluster.cc

```diff
@@ -0,0 +1,35 @@
+#include "DataFormats/SiStripDetSetCompressedCluster/interface/SiStripDetSetCompressedCluster.h"
+
+SiStripDetSetCompressedCluster::SiStripDetSetCompressedCluster() {
+  compressedAmplitudes_.clear();
+  firstStrip_.clear();
+  //error_x_.clear();
+}
+
+SiStripDetSetCompressedCluster::SiStripDetSetCompressedCluster(std::vector<std::pair<uint16_t, bool>>& firstStripMerged,
+                                                               std::vector<float>& errx,
+                                                               std::vector<uint8_t>& inVect) {
+  compressedAmplitudes_.clear();
+  firstStrip_.clear();
+  // error_x_.clear();
+
+  for (auto itFs : firstStripMerged)
+    this->push_back_firstStip(itFs.first, itFs.second);
```

Comment:
Here and below `this->` is not needed, and it doesn't help making the code more readable.

Reply:
it would be more efficient to `reserve()` the full vector size instead of letting it be resized by `push_back()`